stixcore/products/level0/housekeeping.py

```diff
@@ -0,0 +1,204 @@
+"""
+House Keeping data products
+"""
+from stixcore.datetime.datetime import DateTime
+from stixcore.products.level0.quicklook import QLProduct
+from stixcore.products.product import Control, Data
+
+__all__ = ['MiniReport', 'MaxiReport']
+
+
+class MiniReport(QLProduct):
+    """
+    Mini house keeping reported during start up of the flight software.
+    """
+    def __init__(self, *, service_type, service_subtype, ssid, control, data, **kwargs):
+        super().__init__(service_type=service_type, service_subtype=service_subtype,
+                         ssid=ssid, control=control, data=data, **kwargs)
+        self.name = 'mini'
+        self.level = 'L0'
+        self.type = 'hk'
+
+    @classmethod
+    def from_levelb(cls, levelb):
+        packets = super().from_levelb(levelb)
+
+        service_type = packets.get('service_type')[0]
+        service_subtype = packets.get('service_subtype')[0]
+        ssid = packets.get('pi1_val')[0]
+
+        control = Control()
+        control['scet_coarse'] = packets.get('scet_coarse')
+        control['scet_fine'] = packets.get('scet_fine')
+        control['integration_time'] = 0
+        control['index'] = range(len(control))
+
+        # Create array of times as dt from date_obs
+        times = [DateTime(ct, ft).as_float() for ct, ft in control['scet_coarse', 'scet_fine']]
+
+        # Data
+        data = Data()
+        data['time'] = times
+        data['sw_running'] = packets.get_value('NIXD0021', attr='value')
+        data['instrument_number'] = packets.get_value('NIXD0022', attr='value')
+        data['instrument_mode'] = packets.get_value('NIXD0023', attr='value')
+        data['hk_dpu_pcb_t'] = packets.get_value('NIXD0025')
+        data['hk_dpu_fpga_t'] = packets.get_value('NIXD0026')
+        data['hk_dpu_3v3_c'] = packets.get_value('NIXD0027')
+        data['hk_dpu_2v5_c'] = packets.get_value('NIXD0028')
+        data['hk_dpu_1v5_c'] = packets.get_value('NIXD0029')
+        data['hk_dpu_spw_c'] = packets.get_value('NIXD0030')
+        data['hk_dpu_spw0_v'] = packets.get_value('NIXD0031')
+        data['hk_dpu_spw1_v'] = packets.get_value('NIXD0032')
+        data['sw_version'] = packets.get_value('NIXD0001')
```

Comment:
i am not sure but is it possible that this values here are already converted into engineering values?

so fare we only used times and this is ok for level0.
but my understanding was that we will for this voltages.... store the the raw value in level0 and later on in level1 convert it to engineering

Reply:
Yea so with the current setup they are converted here already but as you note this isn't the plan